pages/common/virt-sparsify.md

```diff
@@ -0,0 +1,13 @@
+# virt-sparsify
+
+> libguestfs tool to to make a virtual machine drive images thin-provisioned.
+> Use only for offline machines to avoid data corruption.
+> Home page: <https://libguestfs.org/>.
+
+- Create sparsified comporessed image without snapshots from unsparsified:
```

Comment:
Typo, and maybe this will read better :-)
```suggestion
- Create a sparsified compressed image without snapshots from an unsparsified image:
```

Reply:
Oh, indeed, thank you!